docs/source/configuration/settings-reference.md

```diff
@@ -487,10 +487,26 @@ showRelatedItems
     ```{versionadded} 18.5.0
     ```
 
-
 showTags
     If true, the `Tags` component will show tags from the `subjects` field. Default: true.
 
+isMultilingual
+    If true, features for translating content to multiple languages are available. Default: false.
+
+    ```{deprecated} Volto 19
+    This setting was removed in Volto 19. Instead, Volto checks whether the backend has multilingual features enabled.
```

Comment:
```suggestion
    ```{versionremoved} Volto 19
    This setting was removed in Volto 19.
    Instead, Volto checks whether the backend has multilingual features enabled.
```